vNext/samples/ODataSample.Web/Controllers/CustomersController.cs

```diff
@@ -26,9 +26,22 @@ public IEnumerable<Customer> Get()
 
         // GET api/Customers/5
         [HttpGet("{id}")]
-        public IActionResult Get(int id)
+        public IActionResult Get(int customerId)
         {
-            var customer = _sampleContext.FindCustomer(id);
+            var customer = _sampleContext.FindCustomer(customerId);
+            if (customer == null)
+            {
+                return HttpNotFound();
+            }
+
+            return new ObjectResult(customer);
+        }
+
+        // GET api/Customers/5
```

Comment:
Corrected comment to reflect method: // GET api//FindCustomersWithProduct(productId=1)